AppLovin MAX Demo App - Swift/AppLovin MAX Demo App - Swift/MAX/Native Ads/ALMAXManualNativeAdViewController.swift

```diff
@@ -111,6 +111,10 @@ extension ALMAXManualNativeAdViewController: MANativeAdDelegate
         logCallback()
     }
     
+    func didImpressionNativeAd(_ ad: MAAd) {
```

Comment:
When building at AppLovin - please take care to look at surrounding code. You can see that all methods here are implemented with the `{` on a new line, but yours is not. Also, let's update the ordering of the implementation of the API as well.